src/user/configure-metadata.md

```diff
@@ -4074,12 +4074,14 @@ sampled. For example:
 
 8.  Select a **Period type**.
 
-9.  Assign one or more organisation unit levels. The output value will be
+9.  Assign one or more **organisation unit levels**. The output value will be
     assigned to an organisation unit at this level (or these levels).
-    The input values will come from the organisation unit to which the
+    For **aggregate** data, the input values depends on the selection below; **organisation units providing data**.
+    For **tracker** based data,  the input values will come from the organisation unit to which the
     output is assigned, or from any level lower under the output
     organisation unit.
 
+10. **Organisation units providing data** controls where the input values comes from for **aggregate** data. If "at selected levels only", only organisation units at the selected levels are included. If "At selected levels and all levels below" is selected, organisation units at the selected level(s) and all organisation units below are also included.
 10. Create a **Generator**. The generator is the expression that is used to calculate the predicted value.
```

Comment:
Thanks, looks great. I suggest you increment the numbers for the old points 10 - 15.